index.html

```diff
@@ -11,6 +11,7 @@
     />
 </head>
 <bodybr>
+    <a href="./documenation/documentation.html">Documenatio</a><br/>
```

Comment:
typo: 
it should be Documentation I guess

Reply:
Oh. My bad. Thanks.
Thanks for looking into code